subworkflows/local/taxonomy/main.nf

```diff
@@ -32,15 +35,26 @@ workflow TAXONOMY {
         } else {
             subset_ch = SUBSET_READS_PAIRED(reads_ch, params.read_fraction)
         }
+
+         // Deduplicate reads (if applicable)
```

Comment:
What's the thinking behind making this optional? Do you have a case in mind where we wouldn't want to do this?